### DISCUSSION TITLE
Error: 'NoneType' object has no attribute 'sparkContext'

### DISCUSSION THREAD

**Discussion by ashwardsimpson** [moj-analytical-services/splink#1484] [Q&A] [answered]
I'm trying to run one of the example scripts, changing only what I need to:
https://moj-analytical-services.github.io/splink/demos/examples/spark/deduplicate_1k_synthetic.html

This is the error:
![Screenshot_2023-07-28_14-45-45](https://github.com/moj-analytical-services/splink/assets/92102738/da852423-8697-44d4-bf09-3971d988f379)

And these are my settings:
![Screenshot_2023-07-28_14-46-19](https://github.com/moj-analytical-services/splink/assets/92102738/e0b913f7-3c15-4152-a474-918bead85e81)
![Screenshot_2023-07-28_14-46-40](https://github.com/moj-analytical-services/splink/assets/92102738/af4fcbde-4743-4b5e-b20f-28fda8b39be9)

**Comment by RobinL** [comment, marked as answer]
I think this is the same problem as https://github.com/moj-analytical-services/splink/discussions/1461

Do the suggestions there help?

**Reply by ashwardsimpson** [reply]
that does thanks!
